packages/dds/tree/src/test/testTrees.ts

```diff
@@ -318,6 +326,7 @@ export const testTrees: readonly TestTree[] = [
 			rootFieldSchema: {
 				kind: FieldKinds.required.identifier,
 				types: new Set([allTheFieldsName]),
+				persistedMetadata: undefined,
```

Comment:
Given that test tree now have persistedMetadata, it seems like they should include a case which uses it.